ci/Conformance.md

```diff
@@ -0,0 +1,67 @@
+# Conformance results
+
```

Comment:
Say something like install sonobouy and have the configuration file in place.  Then validate with kubectl.